### PR TITLE
Updated labs 191 and 201 - K8S Version 1.20 

### PR DESCRIPTION
**Pull request by claubota**
*Description of changes:*

Updated labs 191 and 201. Tested both on versions 1.19 and 1.20

Lab 191 - Encrypting Secrets with KMS:

- Fixed typos / rephrased sentences
- Updated reasons to use custom key store to match the documentation

Lab 201 - Resource Management:

- Fixed typos / rephrased sentences
- Reordered pod creation to make it easier to follow
- Fixed output rendering error
- Added output for Resource Quotas

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/beginner/191_secrets/kms-custom-keystore.md [hunk 1]
```diff
@@ -46,18 +44,18 @@ Keep in mind that the KMS Custom Key Store functionality makes use of a minimum
 
 #### Cost
 
-Aside from compliance considerations, your team will want to consider the cost of using this feature. For comparison, I will list the cost of using a CMK created with the default KMS functionality. Then, I will list of the cost of using a CMK created with the custom key store functionality.
+Aside from compliance and security requirements, your may want to consider the cost of using custom key stores. Below you can find a cost comparison between default AWS KMS key store and AWS KMS custom key store.
```
Comment (buzzsurfr) [inline content/beginner/191_secrets/kms-custom-keystore.md:47]: "your may" -> "you may"

Also add a note that this assumes a certain region and provide a link where they can find the latest KMS pricing information (to https://aws.amazon.com/kms/pricing/).

**Review by buzzsurfr — CHANGES_REQUESTED** [review]
Small note additional and change but otherwise LGTM

**Comment by claubota** [comment]
Fixed typos, added pricing information and updated last review date

File: content/intermediate/201_resource_management/basic-pod-limits.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
 ---
 title: "Basic Pod CPU and Memory Management"
-date: 2020-06-22T00:00:00-03:00
+date: 2021-11-10T00:00:00-03:00
 weight: 10
 draft: false
 ---
 
 We will create four pods:
```
Comment (buzzsurfr) [inline content/intermediate/201_resource_management/basic-pod-limits.md:8]: This feels out of place with the **Deploy Metrics Server** section. Consider putting metrics server on it's own page as a module prerequisite, or at least moving above deploying the pods.

File: content/intermediate/201_resource_management/cleaning.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
```
Comment (buzzsurfr) [inline content/intermediate/201_resource_management/cleaning.md:1]: Please rename the _file_ from `cleaning.md` to `cleanup.md` to match with other modules.

**Review by buzzsurfr — CHANGES_REQUESTED** [review]
One small change, and one additional maintenance action (while reviewing the module), then ready for approval.

**Comment by claubota** [comment]
Added a separate page for the Metric Server. Renamed cleaning.md to cleanup.md

**Review by buzzsurfr — APPROVED** [review]
LGTM :shipit:

**Pull request merged**